client/src/components/RatingsAndReviews/ReviewsList.jsx

```diff
@@ -59,11 +70,43 @@ const ReviewsList = class extends React.Component {
     this.setState({ display: reviewsCopy });
   }
 
+  sortClickHandler(value) {
```

Comment:
This was a great approach! I was super curious how the click on a sorting category was going to communicate with the API